### PR TITLE
Feedback

### PR DESCRIPTION
**Pull request by mariano-aguero**
- if a transaction was successful update the list of borrow and supply
- Add snackbar with a demo message
- Add link to the total supply info

### CODE REVIEW

**Comment by mariano-aguero** [comment]
Peek 23-02-2021 08-47

File: src/components/borrow/BorrowList.vue [hunk 1]
```diff
@@ -87,6 +88,11 @@ export default {
   async created() {
     // get all markets
     this.markets = await this.$middleware.getMarkets(this.account)
+    this.$store.subscribe((mutation) => {
+      if (mutation.type === constants.SNACK_SET_SUCCESS_TX) {
+        this.reloadItems()
+      }
+    })
```
Comment (mariano-aguero) [inline src/components/borrow/BorrowList.vue:95]: We subscribed to the store, and checked for the success mutation related to the snackbar

File: src/components/supply/SupplyList.vue [hunk 2]
```diff
@@ -84,6 +85,11 @@ export default {
   async created() {
     // get all markets
     this.markets = await this.$middleware.getMarkets(this.account)
+    this.$store.subscribe((mutation) => {
+      if (mutation.type === constants.SNACK_SET_SUCCESS_TX) {
+        this.reloadItems()
+      }
+    })
```
Comment (mariano-aguero) [inline src/components/supply/SupplyList.vue:92]: Idem here, but for the supply list

File: src/views/MyActivity.vue [hunk 3]
```diff
@@ -1,5 +1,11 @@
 <template>
   <div class="my-activity">
+    <v-snackbar v-model="snackbar" :timeout="3000" app center top color="primary" text>
+      The site is still on beta
+      <template v-slot:action="{ attrs }">
+        <v-btn color="indigo" text v-bind="attrs" @click="snackbar = false"> Close </v-btn>
+      </template>
+    </v-snackbar>
```
Comment (mariano-aguero) [inline src/views/MyActivity.vue:8]: Here is the snackbar for the demo message, maybe we can improve the message and the timeout

File: src/views/MyActivity.vue [hunk 4]
```diff
@@ -60,7 +66,11 @@
                       <v-divider />
                     </v-row>
                     <v-row v-for="(item, index) in totalSupplied" :key="`totalSupplied-${index}`">
-                      <v-col cols="6" class="px-0"><h4>Total Supplied:</h4></v-col>
+                      <v-col cols="6" class="px-0">
+                        <router-link :to="{ name: 'SupplyBorrow' }">
+                          <h4 :style="{ textDecoration: 'underline' }">Total Supplied:</h4>
+                        </router-link>
+                      </v-col>
```
Comment (mariano-aguero) [inline src/views/MyActivity.vue:73]: The link in the total supply to see the tokens

File: src/components/borrow/BorrowList.vue (hunk 1 above)
Comment (pmprete) [reply src/components/borrow/BorrowList.vue:95]: Missing unsubscribe https://vuex.vuejs.org/api/#subscribe

File: src/components/supply/SupplyList.vue (hunk 2 above)
Comment (pmprete) [reply src/components/supply/SupplyList.vue:92]: Same here, missing unsubscribe https://vuex.vuejs.org/api/#subscribe

**Review by pmprete — CHANGES_REQUESTED** [review]
Please add the unsubscribe on component beforeDestroy

File: src/views/MyActivity.vue (hunk 3 above)
Comment (galimba) [reply src/views/MyActivity.vue:8]: `This website is still in BETA. This means rLending is in a testing phase and it is likely to contain errors. <a href="terms">Read more about this notice.</a>`

**Comment by mariano-aguero** [comment]
> Please add the unsubscribe on component beforeDestroy

Done!

File: src/components/supply/SupplyList.vue (hunk 2 above)
Comment (mariano-aguero) [reply src/components/supply/SupplyList.vue:96]: Done!

File: src/components/borrow/BorrowList.vue (hunk 1 above)
Comment (mariano-aguero) [reply src/components/borrow/BorrowList.vue:99]: Done!

File: src/views/MyActivity.vue (hunk 3 above)
Comment (mariano-aguero) [reply src/views/MyActivity.vue:10]: Just, to take into account with the text, the terms section doesn't exist yet

**Review by pmprete — APPROVED** [review]
LGTM remember to modify the Beta message later on

**Pull request merged**